app/quality-check/get-quality-checks-count.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../data')
+
+const getQualityChecksCount = async () => {
+  return db.paymentRequest.count()
```

Comment:
`db.qualityCheck`

Reply:
Include tests for all get-x-counts in integration/local

Reply:
Test for "get-x-counts" added